practice_app/api/views.py

```diff
@@ -1,3 +1,66 @@
-from django.shortcuts import render
+import requests
+import re
+from django.http import JsonResponse
+from . import api_keys
 
-# Create your views here.
+
+
+def google_scholar(request):
+    number = request.GET.get("rows")
+    search = request.GET.get("title")
+    if search == None or search == "":
+        return JsonResponse({'status': 'Title to search must be given.'}, status=404)
+    if number == None or number == "" or not number.isnumeric():
+        number = 5
+    else:
+        number = int(number)
+
+    request = requests.get('https://serpapi.com/search.json?engine=google_scholar&q=' + search + '&hl=en&num=' + str(number) + '&api_key=' + api_keys.api_keys['serp_api'])
+    if request.status_code == 200:
+        request = request.json()
+        papers = request['organic_results']
+        response = {}
+        results = []
+        for paper in papers:
+            pub_info = paper['publication_info']
+            paper_info = {}
+            paper_info['source'] = 'google_scholar'
+
+            if 'authors' in  pub_info.keys():
```

Comment:
I suppose we decided to remove author field

Reply:
We decided to decide it later.